tests/providers/amazon/aws/operators/test_batch.py

```diff
@@ -211,3 +218,114 @@ def test_execute(self, mock_conn):
             computeResources=compute_resources,
             tags=tags,
         )
+
+
+def create_context(task, dag=None):
```

Comment:
whats the difference between the tests here and the ones under `deferrable/hooks/test_batch_client.py` ?

Reply:
`deferrable/hooks/test_batch_client.py` are hook tests and the `tests/providers/amazon/aws/operators/test_batch.py' are for operators.